ibis/expr/window.py

```diff
@@ -71,7 +102,13 @@ def __init__(
                 x = ops.SortKey(x).to_expr()
             self._order_by.append(x)
 
-        self.preceding = _sequence_to_tuple(preceding)
+        if isinstance(preceding, RowsWithMaxLookback):
+            self.preceding = preceding.rows
+            self.max_look_back = preceding.max_lookback
```

Comment:
Can we call this `max_lookback` instead of `max_look_back` to be consistent with the rest of the implementation?

Reply:
Yes, fixed